test2.log

```diff
@@ -0,0 +1,5 @@
+Script started on 2024-04-27 13:50:15+05:30 [TERM="xterm-256color" TTY="/dev/pts/0" COLUMNS="100" LINES="18"]
```

Comment:
Please clean up the PR so it doesn't contain files that shouldn't be merged.

Reply:
Thank you for pointing out, i will do as you said.